lxd/patches.go

```diff
@@ -164,6 +165,47 @@ func patchesApply(d *Daemon, stage patchStage) error {
 }
 
 // Patches begin here
+// renames any config incorrectly set entries due to the lvm.thinpool_name typo
+func patchThinpoolTypoFix(name string, d *Daemon) error {
+
+	tx, err := d.cluster.Transaction(func(tx *db.ClusterTx) error {
+
+		// Fetch the IDs of all existing nodes.
+		nodeIDs, err := query.SelectIntegers(tx, "SELECT id FROM nodes")
+
+		// Fetch the IDs of all existing lvm pools.
+		poolIDs, err := query.SelectIntegers(tx, "SELECT id FROM storage_pools WHERE driver='lvm'")
+
+		for _, poolID := range poolIDs {
+			// Fetch the config for this lvm pool and check if it has the
+			// lvn.thinpool_name key.
+			config, err := query.SelectConfig(
+				tx, "storage_pools_config", "storage_pool_id=?", poolID)
+
+			value, ok := "lvm.thinpool"
```

Comment:
What's this supposed to do?

Did you mean it to be `value, ok := config["lvm.thinpool"]` perhaps?